PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -428,15 +482,13 @@ function Get-PrivilegedRole {
         $TenantHasPremiumLicense
     )
 
-    Write-Information "Entering"
     $PrivilegedRoles = [ScubaConfig]::ScubaDefault('DefaultPrivilegedRoles')
```

Comment:
Crutch, can the config value for DefaultPrivilegedRoles be overridden by a custom config file created by the user?